src/Resources/AndroidConfig.php

```diff
@@ -187,7 +187,7 @@ public function toArray(): array
     {
         return [
             'collapse_key' => $this->getCollapseKey(),
-            'priority' => ! is_null($this->getPriority()) ? $this->getPriority()->getValue() : null,
+            'priority' => ! is_null($this->getPriority()) ? $this->getPriority()->label : null,
```

Comment:
Could we do this and cover both?
```suggestion
            'priority' => ! is_null($this->getPriority()) ? $this->getPriority()->label ?? $this->getPriority()->getValue() : null,
```

Reply:
I only did a quick test in Tinkerwell using spatie/enum ^2.3, but that did work 👍  PR has been amended

Reply:
Merged and tagged, thanks!